internal/packages/archetype/package.go

```diff
@@ -92,6 +94,14 @@ func createPackageInDir(packageDescriptor PackageDescriptor, cwd string) error {
 
 	}
 
+	if len(packageDescriptor.ExcludeChecks) > 0 {
```

Comment:
There were no exclude checks when creating a content package. These are the files created:
```
 $ elastic-package create package 
Create a new package
? Package type: content
? Package name: new_package
? Version: 0.0.1
? License: Elastic-2.0
? Package title: New Package
? Description: This is a new package.
? Categories: custom
? Kibana version constraint: ^8.15.1
? Required Elastic subscription: basic
? Github owner: elastic/integrations
? Owner type: elastic
New package has been created: new_package
Done

 $ cd new_package

 $ ls -l 
total 20
-rw-r--r-- 1 user user  226 sep  9 18:22 changelog.yml
drwxr-xr-x 2 user user 4096 sep  9 18:22 docs
drwxr-xr-x 2 user user 4096 sep  9 18:22 img
-rw-rw-r-- 1 user user 3860 sep  9 18:22 LICENSE.txt
-rw-r--r-- 1 user user  560 sep  9 18:26 manifest.yml
```

Reply:
That's correct. I added this mainly for testing by now.